book/history.md

```diff
@@ -175,7 +190,7 @@ this codebase). Likewise, the
 originally developed by Netscape starting in 1997; the
 [Firefox](https://en.wikipedia.org/wiki/Firefox) browser is descended from this
 codebase.  During the first browser war period, nearly all of the core features
-you will implement in your browser that goes along with this book were added,
+of the simple browser that goes along with this book were added,
 including CSS, DOM, and JavaScript.
```

Comment:
```suggestion
of this book's simple browser were added,
including CSS, DOM, and JavaScript.
```

Otherwise kind of confusing to parse. Maybe rephrasing some other way is better.